engineering/conventions-go.md

```diff
@@ -62,4 +57,10 @@ A blank line is usually added after the end of any control structure, too.
 
 ### Naming
 
-* Prefer full names (e.g. `Repository`, not `Repo`).
\ No newline at end of file
+* Prefer full names (e.g. `Repository`, not `Repo`).
+
+## CLI
+
+* Put sources for your executable commands in `cmd/<command-name>/main.go`.
```

Comment:
Do you mean with this line that all the code must be inside `main.go` or maybe `cmd/<command-name>/main.go` should be `cmd/<command-name>/` ?